plugins/plugin-chart-echarts/src/BoxPlot/transformProps.ts

```diff
@@ -162,9 +162,8 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
           },
         },
       },
-      // @ts-ignore
       ...outlierData,
-    ],
+    ] as SeriesOption,
```

Comment:
I would prefer not to cast this but rather `@ts-ignore` any specific typing problems that absolutely can't be avoided. I think this needs to be done separately like so:

```typescript
const series: BoxplotSeriesOption = [...];
```
and then later
```typescript
const echartsOptions: EChartsOption = {
  ...,
  series,
  ...,
}
```

Reply:
done for box and pie chart